### DISCUSSION TITLE
Vs. Diffusers

### DISCUSSION THREAD

**Discussion by billnye2** [finegrain-ai/refiners#406] [Q&A] [answered]
Why use this instead of diffusers?

Thank you

**Comment by limiteinductive** [comment, marked as answer]
Hey @billnye2!

Thank you for your question. To understand Refiners, you should read the [Manifesto](https://refine.rs/home/why/), which is available in our docs. To dive deeper into what makes Refiners unique, look at [this blog post](https://blog.finegrain.ai/posts/simplifying-ai-code/) I wrote.

TLDR: Although diffusers shares some functional similarities, Refiners is not restricted to diffusion. For example, we support other foundation models like SAM. Our particularity is that Refiners is a micro-framework focusing on implementing Adapters on foundational models.

Best of luck with your project!
